pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -851,8 +849,13 @@ protected void handleProducer(final CommandProducer cmdProducer) {
                                 // until the previous producer creation
                                 // request
                                 // either complete or fails.
-                                ServerError error = !existingProducerFuture.isDone() ? ServerError.ServiceNotReady
-                                        : getErrorCode(existingProducerFuture);
+                                ServerError error = null;
+                                if(!existingProducerFuture.isDone()) {
+                                    error = ServerError.ServiceNotReady;
+                                }else {
```

Comment:
formatting is incorrect

Also since you are gonna fix the formatting do you mind inverting the if condition to put `if (existingProducerFuture.isDone())` first -> Easier to read.